integration-test/test-cookie.js

```diff
@@ -53,11 +41,11 @@ describe('Cookie protection tests', () => {
         expect(result.expires).toBeLessThan(Date.now() + 605_000_000)
     })
 
-    it('Erroneous values do not throw', async () => {
-        const page = await browser.newPage()
-        await page.goto('http://localhost:8080/index.html')
-
+    test('Erroneous values do not throw', async ({ page }) => {
+        await gotoAndWait(page, '/index.html')
         const result = await page.evaluate(async () => {
+            document.cookie = 'a=b; expires=Wed, 21 Aug 2040 20:00:00 UTC;'
```

Comment:
Is this needed here? There's a test above for it

Reply:
yeah it's needed because this test looks for 'a'. it worked before because the harness setup was leaking state across tests.